R/pkg/R/sparkR.R

```diff
@@ -362,6 +362,10 @@ sparkR.session <- function(
   enableHiveSupport = TRUE,
   ...) {
 
+  if (length(sparkConfig[["spark.sql.warehouse.dir"]]) == 0) {
+    sparkConfig[["spark.sql.warehouse.dir"]] <- tempdir()
```

Comment:
I think Shivaram is talking about a spark property, not a parameter (if perhaps your camel casing is an indication)
Basically, we don't want to change `spark.sql.warehouse.dir` because it could be set already in an earlier point (just not accessible here)